packages/ui/src/RadioGroup.module.scss

```diff
@@ -0,0 +1,12 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.radioGroup {
+  display: flex;
+  flex-direction: column;
+
+  // Error state
+  &.error label > input:checked ~ * {
```

Comment:
We are kind of violating the incapsulation of the `Radio` component here. Building on the context idea (https://github.com/hazelcast/frontend-shared/pull/54/files#r517947676), couldn't we pass the `error: boolean` to the context and make the `Radio` style the error state internally based on the prop?